data/ledger.go

```diff
@@ -81,7 +81,7 @@ type roundSeed struct {
 func LoadLedger[T string | ledger.DirsAndPrefix](
 	log logging.Logger, dir T, memory bool,
 	genesisProto protocol.ConsensusVersion, genesisBal bookkeeping.GenesisBalances, genesisID string, genesisHash crypto.Digest,
-	blockListeners []ledgercore.BlockListener, cfg config.Local,
+	cfg config.Local,
```

Comment:
To make sure I understand, the `blockListeners` arg was never passed in with a nonzero slice, so you removed it?

Reply:
that's correct. I think originally `data.Ledger` used `node` as the only listener, then txPool was extended to require a listener as well, and this was refactored to set `node` and `txPool` as listeners after txPool construction since it requires ledger on initialization.